plugins/node/opentelemetry-instrumentation-koa/examples/src/client.ts

```diff
@@ -1,9 +1,13 @@
 'use strict';
 
 // eslint-disable-next-line import/order
-const tracer = require('./tracer')('example-koa-client');
-const api = require('@opentelemetry/api');
-const axios = require('axios').default;
+// const tracer = require('../tracer')('example-koa-client');
+import { setupTracing } from "./tracer";
+const tracer = setupTracing('example-express-client');
+// const api = require('@opentelemetry/api');
```

Comment:
any reason to keep the `require` version as comments? (here and in other places)